xbmc/addons/kodi-addon-dev-kit/include/kodi/AddonBase.h

```diff
@@ -320,8 +329,14 @@ class ATTRIBUTE_HIDDEN CAddonBase
     return ADDON_STATUS_UNKNOWN;
   }
 
+  virtual ADDON_STATUS CreateInstanceEx(int instanceType, std::string instanceID, KODI_HANDLE instance, KODI_HANDLE& addonInstance, const char *version)
```

Comment:
Here you have still the "const char *" in IAddonInstance it is it std::string :)